ignition/boot-process.md

```diff
@@ -4,25 +4,25 @@ The Container Linux startup process is built on the standard [Linux startup proc
 
 ## Bootloader
 
-[GRUB][grub] is the first program executed when a Container Linux system boots.
+[GRUB][grub] is the first program executed when a Container Linux system boots. The Container Linux [GRUB config][grub config] has several roles.
 
-The Container Linux [GRUB config][grub config] has several roles. First, the GRUB config [specifies which `usr` partition to use][gptprio.next] from the two `usr` partitions Container Linux uses to provide atomic upgrades and rollbacks.
+First, the GRUB config [specifies which `usr` partition to use][gptprio.next] from the two `usr` partitions Container Linux uses to provide atomic upgrades and rollbacks.
 
-Additionally, GRUB [checks for a file called `coreos/first_boot` in the EFI System Partition][check-file] to determine if this is the first time a machine has booted. If that file is found, GRUB sets the `coreos.first_boot=detected` Linux kernel command line parameter. This parameter is used in later stages of the boot process.
+Second, GRUB [checks for a file called `coreos/first_boot` in the EFI System Partition][check-file] to determine if this is the first time a machine has booted. If that file is found, GRUB sets the `coreos.first_boot=detected` Linux kernel command line parameter. This parameter is used in later stages of the boot process.
 
 Finally, GRUB [searches for the initial disk GUID][search-guid] (00000000-0000-0000-0000-000000000001) built into Container Linux images. This GUID is randomized later in the boot process so that individual disks may be uniquely identified. If GRUB finds this GUID it sets another Linux kernel command line parameter, `coreos.randomize_guid=00000000-0000-0000-0000-000000000001`.
 
 ## Early user space
 
-The next major milestone in the Container Linux startup process is the jump into the initial RAM file system. In addition to its standard responsibility of mounting the root filesystem, the initramfs is also where the disk GUID is randomized and where Ignition runs.
+After GRUB, the Container Linux startup process moves into the initial RAM file system. The initramfs mount the root filesystem, randomize the disk GUID, and run Ignition.
```

Comment:
"initramfs" is a singular noun despite the trailing "s" (Initial RAM filesystem).